cineast-core/src/main/java/org/vitrivr/cineast/core/iiif/IIIFConfig.java

```diff
@@ -0,0 +1,190 @@
+package org.vitrivr.cineast.core.iiif;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import org.vitrivr.cineast.core.iiif.imageapi.ImageApiVersion;
+
+/**
+ * IIIF configuration used to fetch media files from remote servers.
+ *
+ * @author singaltanmay
+ * @version 1.0
+ * @created 28.05.21
```

Comment:
Please avoid the usage of @version and @created tags in javadoc, i know we have them sometimes in the codebase, but we're trying to get rid of them :)

Reply:
I have removed @author @version and @created tags from all the Javadoc I have created